test/functional/plural/plural.js

```diff
@@ -87,6 +103,26 @@ QUnit.test( "should return plural form", function( assert ) {
 	assert.equal( Globalize( "ar" ).plural( 199 ), "many" );
 	assert.equal( Globalize( "ar" ).plural( 3.14 ), "other" );
 
+	assert.equal( Globalize( "ar" ).plural( 0, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 1, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 2, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 3, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 6, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 9, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 10, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 11, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 15, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 21, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 70, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 99, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 100, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 101, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 102, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 103, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 111, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 199, { type: "ordinal" } ), "other" );
+	assert.equal( Globalize( "ar" ).plural( 3.14, { type: "ordinal" } ), "other" );
```

Comment:
What do these assertion accomplish when the expected value is always "other"? If this is useful at all, you might as well loop create a pluralFormatter, then loop over a number range and assert the ordinal for each.